src/oskbd/mod.rs

```diff
@@ -59,3 +60,24 @@ impl KeyEvent {
         Self { code, value }
     }
 }
+
+#[derive(Debug, Clone, Copy)]
+pub enum ScrollEventKind {
```

Comment:
Can we eliminate this and move all of the logic inside of the Linux event loop file - this doesn't apply to Windows. Same thing with the `HiRes` variants of `OsCode`. The Linux event loop should have sufficient info to differentiate without needing the extra OsCodes or this enum..

Reply:
Can't we move this to a linux-only file instead of removing? 

> The Linux event loop should have sufficient info to differentiate without needing the extra OsCodes or this enum.

yeah, but at this point 
https://github.com/jtroo/kanata/blob/e381af017924cf960968c5ef491e85a066ca3fd2/src/kanata/linux.rs#L60
getting scroll type seems to be the easiest done from OsCode, while being equally efficient to the other options

Reply:
If you have any particular changes on mind, push them directly to this branch, otherwise it looks like we're gonna play cat and mouse game back and forth